model/logs/mobile-events.yaml

```diff
@@ -1,17 +1,76 @@
 groups:
-  - id: ios.lifecycle.events
```

Comment:
Shouldn't we move this to deprecated? As now we don't want to ever remove things?

Reply:
Great point! Unfortunately we don't have a CI check for event removal (yet).

Since we're not removing metrics or attributes anymore, we should do the same for events and deprecate them instead - if someone generates code for event names, they would be broken with this change.

Reply:
Added as deprecated